xarray/backends/api.py

```diff
@@ -318,6 +318,33 @@ def maybe_decode_store(store, lock=False):
     return maybe_decode_store(store)
 
 
+def open_rasterio(filename, add_latlon=False):
```

Comment:
Should the crs argument be passed along to `add_latlon_coords_from_crs`? I.e., 

```
def open_rasterio(filename, add_latlon=False, crs=None)
```

or


```
def open_rasterio(filename, add_latlon=False, **kwargs)
```